cmp/treemap/TreeMap.ts

```diff
@@ -21,7 +21,8 @@ import {errorMessage} from '@xh/hoist/cmp/error';
 import {mask} from '@xh/hoist/cmp/mask';
 import {Highcharts} from '@xh/hoist/kit/highcharts';
 import {wait} from '@xh/hoist/promise';
-import {logError, logWithDebug} from '@xh/hoist/utils/js';
+import {logWithDebug} from '@xh/hoist/utils/js';
```

Comment:
This was surprising - looked like a mistake in this context, then realized that its in our decorators package which is aliased in via `/js`.  

Seems like we might have a choice - either keep this where it is (all decorators come from Decorators.ts) or move it into the new log utils (all log methods come from LogUtils.ts).

Reply:
See PR intro above. I don't think moving any log related decorators into the /log/ packages is a good idea.

Reply:
I would have preferred to leave the logging stuff together, but understand colin's argument! so am good with that.